migrations/Version202311241512382260_taoQtiTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiTest\scripts\install\RegisterTestRunnerPlugins;
+use oat\taoTests\models\runner\plugins\PluginRegistry;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * phpcs:disable Squiz.Classes.ValidClassName
```

Comment:
Since it is already modified :) 

```suggestion
 * phpcs:disable Squiz.Classes.ValidClassName
```